test/sqllogictest/copy_to.slt

```diff
@@ -0,0 +1,46 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Basic tests for `COPY ... TO` statements.
+
+mode cockroach
+
+# Start from a pristine state
+reset-server
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_copy_to_expr = on;
+----
+COMPLETE 0
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_aws_connection = on;
+----
+COMPLETE 0
+
+statement ok
+CREATE CONNECTION aws_conn TO AWS (ASSUME ROLE ARN 'assume-role', ASSUME ROLE SESSION NAME 'session-name');
+
+simple conn=mz_system,user=mz_system
+COPY (VALUES (1, 2, 3)) TO 's3://path' WITH (AWS CONNECTION = aws_conn, FORMAT = 'csv', MAX FILE SIZE = '100MB')
+----
+COMPLETE 1
```

Comment:
For some reason sqllogic is returning a `COMPLETE` instead of a `COPY` tag (while pgwire tests above return the `COPY` tag). Is this expected?

Reply:
An `INSERT` statement also returns a `COMPLETE` tag, so looks like this is expected.